src/tacacs/pam/Makefile

```diff
@@ -20,6 +20,7 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	git apply ../0004-management-vrf-support.patch
 	git apply ../0005-pam-Modify-parsing-of-IP-address-and-port-number-to-.patch
 	git apply ../0006-Add-support-for-source-ip-address.patch
+	git apply ../0007-handle-bad-password-set-by-sshd.patch
```

Comment:
Keep it as 0010 as the master commit? Then later if we decide to backport more patches, others will be easier.

Reply:
Not really. As master has diverged well with 3 patches, this patch had to be manually re-created. The contents of 0010 & 0007 are different but for the same purpose. Similarly, in future, if we would need any patch from master it has to be re-created manually. The title of the patch would indicate the patch purpose, which is same in both master & here.

Reply:
Sorry for misleading. Is it possible to keep original file name `../0010-handle-bad-password-set-by-sshd.patch`. I am okay if the file content changes.

Reply:
I got your point. 
The number is just meant for sequence of application. So here, this patch has to be applied after 0006, hence 0007.